resources/conf/wdio-bstack-flutterAndroid.conf.js

```diff
@@ -0,0 +1,48 @@
+var defaults = require("./wdio.conf.js");
+var _ = require("lodash");
+
+
+var overrides = {
+  user: process.env.BROWSERSTACK_USERNAME,
+  key: process.env.BROWSERSTACK_ACCESS_KEY,
+  specs: ["./src/test/flutterAndroid.spec.js"],
+  host: "hub.browserstack.com",
+  capabilities: [
+    {
+      maxInstances: 1,
+      "browserstack.debug": true,
+      automationName: "Flutter",
+      os_version: "11.0",
+      realMobile: 'true',
+      device: "Samsung Galaxy S21",
+      app: "bs://<app_id_camera_injection>", //pass the app_url for camera_injection apk placed under /resources/app directory
+      "browserstack.uploadMedia":['media://<media_url>'], //pass the media_url for .png file placed under /resources/image directory
```

Comment:
same as above. uploadMedia also has custom id. please explicitly mention that too.